birdhouse/components/stac/docker-compose-extra.yml

```diff
@@ -52,12 +52,5 @@ services:
       timeout: 5s
       retries: 5
 
-  # extend proxy with endpoint and config for STAC API access
-  proxy:
-    volumes:
-      - ./components/stac/conf.extra-service.d:/etc/nginx/conf.extra-service.d/stac:ro
-    links:
-      - stac
```

Comment:
Not sure if the original intention here was to be a link to `stac-browser` instead? 

At the very least, the following config does a `proxy_pass` that would require some docker link:
https://github.com/bird-house/birdhouse-deploy/blob/stac-quick-fixes/birdhouse/components/stac/config/proxy/conf.extra-service.d/stac.conf.template#L19C20-L19C39

Reply:
links are used to create aliases to other components. From the [docs](https://docs.docker.com/compose/compose-file/compose-file-v3/#links): 

> Links are not required to enable services to communicate - by default, any service can reach any other service at that service’s name. 

They are not required here since the "stac" component is named "stac"

Reply:
Ok. Acceptable as long as no specific networks are defined, since they all use `default`.